### PR TITLE
feature: collapse migrations #177

### PR DESCRIPTION
**Pull request by invzbl3**
develop

## GitHub Board

**Issue link**

[#177](https://github.com/ita-social-projects/dokazovi-be/issues/177)

## Code reviewers
- [x] @melnikovae87

## Summary of issue

- [x] Remove old migration scripts
- [x] Create a script to initialize the database
- [x] Create a script to add demo data

## Summary of change

- [x]  Collapse migration scripts

### CODE REVIEW

File: src/main/resources/db/migration/V1__init_db.sql [hunk 1]
```diff
@@ -1,159 +1,292 @@
...
+			primary key,
+	version varchar(50),
+	description varchar(200) not null,
+	type varchar(20) not null,
+	script varchar(1000) not null,
+	checksum integer,
+	installed_by varchar(100) not null,
+	installed_on timestamp default now() not null,
+	execution_time integer not null,
+	success boolean not null
```
Comment (ad-artem-melnykov) [inline src/main/resources/db/migration/V1__init_db.sql:14]: Usually, the `Flyway` creates this table by itself. Looks like redundant

File: src/main/resources/db/migration/V1__init_db.sql [hunk 2]
```diff
@@ -1,159 +1,292 @@
...
-  EMAIL VARCHAR,
-  PASSWORD VARCHAR,
-  STATUS VARCHAR,
-  FIRST_NAME VARCHAR,
-  LAST_NAME VARCHAR,
-  QUALIFICATION VARCHAR,
-  PHONE VARCHAR,
-  BIO TEXT,
-  CREATED_AT TIMESTAMP DEFAULT (NOW())
+create table roles
```
Comment (ad-artem-melnykov) [inline src/main/resources/db/migration/V1__init_db.sql:22]: Could you choose one SLQ convention for scripts? I prefer to use `UPPERCASE`.

File: src/main/resources/db/migration/V1__init_db.sql [hunk 3]
```diff
@@ -1,159 +1,292 @@
...
-  IS_PRIMARY BOOLEAN
+create table tags
+(
+	tag_id serial not null
+		constraint tags_pkey
+			primary key,
+	tag varchar
+		constraint unique_name
+			unique
 );
```
Comment (ad-artem-melnykov) [inline src/main/resources/db/migration/V1__init_db.sql:71]: Maybe would be better If you apply code format for all scripts? Does it make sense?

**Review by ad-artem-melnykov — CHANGES_REQUESTED** [review]

**Comment by ad-artem-melnykov** [comment]
If you create a PR for merge in a dev/master branch, please fill all correspondent fields related to your PR.
For example:
Title: `PR - #### - Add some functionality` OR `Feature - #### - Add login page` etc.
Description: Leave short information about your PR and remove extras if needed

**Review by melnikovae87 — APPROVED** [review]

**Pull request merged**